Blish HUD/GameServices/Input/Keyboard/KeyBinding.cs

```diff
@@ -10,6 +10,10 @@ namespace Blish_HUD.Input {
     /// Allows for actions to be ran as the result of a provided key combination.
     /// </summary>
     public class KeyBinding {
+        /// <summary>
+        /// Fires when the keys of the <see cref="KeyBinding"/> are changed.
+        /// </summary>
+        public event EventHandler<EventArgs> KeysChanged; 
```

Comment:
Let's name this `BindingChanged`

Reply:
Funny, that was how I named it before I went with KeysChanged. :)